pkg/controller/cloud/node_controller.go

```diff
@@ -313,11 +313,16 @@ func (cnc *CloudNodeController) MonitorNode() {
 }
 
 func (cnc *CloudNodeController) UpdateCloudNode(_, newObj interface{}) {
-	if _, ok := newObj.(*v1.Node); !ok {
+	node, ok := newObj.(*v1.Node)
+	if !ok {
 		utilruntime.HandleError(fmt.Errorf("unexpected object type: %v", newObj))
 		return
 	}
-	cnc.AddCloudNode(newObj)
+
+	cloudTaint := getCloudTaint(node.Spec.Taints)
```

Comment:
@jhorwit2 - this check is done on `addCloudNode`, and we log there if cloud taint isn’t present.

Reply:
Yeah, the problem here is that we excessively log it on every node update event, and node update events happen a lot since every heartbeat counts as an update